src/main/java/gc/cafe/service/ProductService.java

```diff
@@ -0,0 +1,20 @@
+package gc.cafe.service;
+
+import gc.cafe.entity.Products;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface ProductService {
+    //상품 등록
+    //상품 단건조회
+    //상품 전체조회
+    //상품 수정
+    //상품 삭제
+    public byte[] registerProduct(Products products);   //PK(UUID) 반환
+    public Optional<Products> findProduct(byte[] productId);
+    public List<Products> findAllProducts();
+    public byte[] updateProduct(byte[] beforeProductId, Products newProducts);
+    public void removeProduct(byte[] productsId);
```

Comment:
이거는 별거 아니지만 개인적으로는 주석을 이렇게 작성합니다
```suggestion
    //상품 등록
    public byte[] registerProduct(Products products);   //PK(UUID) 반환
    //상품 단건조회
    public Optional<Products> findProduct(byte[] productId);
    //상품 전체조회
    public List<Products> findAllProducts();
    //상품 수정
    public byte[] updateProduct(byte[] beforeProductId, Products newProducts);
    //상품 삭제
    public void removeProduct(byte[] productsId);
```

Reply:
사소하더라도 의견남겨주셔서 감사합니다 미처 신경쓰지 못한 이런 디테일이 쌓여서 기본기가 된다고 생각하고 있습니다 참고하겠습니다!